### PR TITLE
Adding SSD Inception V2 Model for Gestures

### PR DESCRIPTION
**Pull request by fcr3**
This branch contains a folder for a custom SSD Inception V2 model based on the SSD Inception V2 model provided by Tensorflow's Object Detection API. In the folder ncappzoo/tensorflow/SSD_Inception_V2 specifically, there contains a README.md for ease-of-use, a Makefile and AUTHORS file per the standard for additions to the ncappzoo (plus some extra features), a demo python script, a script to download weights for the custom Tensorflow frozen graph (utilized by the Makefile), and a required custom .json config file for the OpenVINO Model Optimizer.

### CODE REVIEW

**Comment by tomepetty** [comment]
Hi Christian,

It looks like the Makefile was hard-coded with the /opt/intel/openvino/deployment_tools/...... path for the CPU extensions. 

Maybe you can set the makefile to check for their OpenVINO environment vars, then you can access the $(INTEL_OPENVINO_DIR) or $(INTEL_CVSDK_DIR) environment variable to access their install location. ex: $(INTEL_OPENVINO_DIR)/deploment_tools/.........

**Comment by nealvis** [comment]
Can you move this project to the ncappzoo/networks directory?  the ncappzoo/tensorflow directory is for legacy networks, eventually they will all probably move to networks directory

File: networks/ssd_inception_v2/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+make run
+```
+The example runs inference using a video stream taken from an existing camera device. Bounding boxes and their associated probabilities and classifications are rendered on frames of the video stream and presented on a OpenCV frame.
+
+
+## Makefile
+Provided Makefile describes various targets that help with the above mentioned tasks.
+
+### make run
+Runs a sample application with the FP32 network.
```
Comment (nealvis) [inline networks/ssd_inception_v2/README.md]: note that this runs on CPU for CPU's that OpenVINO supports.

**Comment by fcr3** [comment]
I am closing this pull request due to the absence of signoffs on previous commits. Apologies for the inconvenience.

**Comment by fcr3** [comment]
Please see duplicate pull request with signoffs. :)

File: networks/ssd_inception_v2/ssd_inception_v2.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+"""
+ Copyright (C) 2018-2019 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (nealvis) [inline networks/ssd_inception_v2/ssd_inception_v2.py]: Be sure all modifications are noted per Apache 2.0 license

**Pull request closed without merging**